src/submitQuest.js

```diff
@@ -45,6 +50,14 @@ export async function submitQuest(isSetUpstream, isListening, environment) {
     process.exit(0);
   }
 
+  const remoteMajorVersion = quest.info.version.match(/^[0-9]+(?=\.)/g);
```

Comment:
For this chunk, can you use the `semver` package? (It's already installed as a dependency). Something like this will be cleaner/more robust.

```javascript
import semver from "semver"

// ...
const diff = semver.diff(quest.info.version, quest.localVersion());
if (diff == "major" || diff == "premajor") { ... }
``